cpp/libcudf_kafka/tests/CMakeLists.txt

```diff
@@ -59,12 +57,7 @@ if("$ENV{CONDA_BUILD}" STREQUAL "1")
     set(CONDA_INCLUDE_DIRS "$ENV{BUILD_PREFIX}/include" "$ENV{PREFIX}/include")
     set(CONDA_LINK_DIRS "$ENV{BUILD_PREFIX}/lib" "$ENV{PREFIX}/lib")
     message(STATUS "Conda build detected, CMAKE_SYSTEM_PREFIX_PATH set to: ${CMAKE_SYSTEM_PREFIX_PATH}")
-elseif(DEFINED ENV{CONDA_PREFIX})
-    set(CMAKE_SYSTEM_PREFIX_PATH "$ENV{CONDA_PREFIX};${CMAKE_SYSTEM_PREFIX_PATH}")
-    set(CONDA_INCLUDE_DIRS "$ENV{CONDA_PREFIX}/include")
-    set(CONDA_LINK_DIRS "$ENV{CONDA_PREFIX}/lib")
-    message(STATUS "Conda environment detected, CMAKE_SYSTEM_PREFIX_PATH set to: ${CMAKE_SYSTEM_PREFIX_PATH}")
```

Comment:
Is there a reason this was removed? I believe this was previously needed to prioritize getting libraries from the conda environment as opposed to a system location.

Reply:
CMake 3.18+ adds `ENV{CONDA_PREFIX}` to `CMAKE_SYSTEM_PREFIX_PATH` ( https://gitlab.kitware.com/cmake/cmake/-/merge_requests/4773/ )

I don't think the CONDA_INCLUDE_DIRS / CONDA_LINK_DIRS are referenced elsewhere.

Reply:
Didn't know this, cool!